### PR TITLE
SPRT-171 change size of images on popups of map

### PR DESCRIPTION
**Pull request by facuzero**
To fix https://github.com/PukaraES/Cicerone/issues/70.

### CODE REVIEW

**Review by olea — COMMENTED** [review]
Can you share example screenshots, please?

**Comment by olea** [comment]
This is a lot of better, but it fit images of any proportion in the same box size. What is needed is to scale the image to fit in the current box keeping the image proportions.

**Comment by njrzr** [comment]
@facuzero what do you need to do here is to wrap the image on a div and set the width and height to the same size(square) and to the image add the utility of object-contain so that the image fits keeping his aspect ratios inside this wrapper.

File: components/WLMap.vue [hunk 1]
```diff
@@ -13,10 +13,10 @@
 
             <v-marker-cluster>
               <l-marker v-for="(data, index) in getData" :key="index" :lat-lng="[getCoords[index][1], getCoords[index][0]]">
-                <l-popup>
-                  <NuxtLink :to="{ name: 'places-id-place-details', params: { id: getEntity(index), index: index } }">
-                    <div class="p-2">
-                      <img :src="data.image != undefined ? data.image.value + '?width=320px' : '/images/default-image.jpg'">
+                <l-popup class="w-40 h-40">
```
Comment (njrzr) [inline components/WLMap.vue:16]: @facu what did you did here? You should have wrapped the img element with a div and add these utilities there, change it.

File: components/WLMap.vue [hunk 2]
```diff
@@ -13,10 +13,10 @@
...
+                  <NuxtLink :to="{ name: 'places-id-place-details', params: { id: getEntity(index), index: index }}">
+                    <div class="p-2 w-30 h-30 flex flex-wrap object-cover">
```
Comment (njrzr) [inline components/WLMap.vue:18]: @facuzero object-cover doesn't go here, you have to add it to the img element, and is object-contain, not object-cover, and add the items-center and justify-center to center the elements.

File: components/WLMap.vue [hunk 3]
```diff
@@ -13,10 +13,10 @@
...
+                      <img :src="data.image != undefined ? data.image.value + '?width=320px' : '/images/default-image.jpg'" :class="['w-32, h-32']">
```
Comment (njrzr) [inline components/WLMap.vue:19]: @facuzero what did you did here? you can add class without the colons, when the colons are used is for when using conditions, or the data object, and you should add here the object-contain.

**Review by njrzr — CHANGES_REQUESTED** [review]
@facuzero read the comment a made, and make the corresponding fixes.

File: components/WLMap.vue (hunk 1 above)
Comment (facuzero) [reply components/WLMap.vue:16]: ok im gonna change

File: components/WLMap.vue (hunk 3 above)
Comment (facuzero) [reply components/WLMap.vue]: @njrzr  I think everything is fixed

**Comment by olea** [comment]
Checking 5a3b4ea3cb82f62f02d135a95e63839fca9e3206 there are differences in sizes:

Captura desde 2022-08-24 19-49-20
Captura desde 2022-08-24 19-49-12
Captura desde 2022-08-24 19-49-05

:-m

**Comment by olea** [comment]
Checked 16a72c15ac5b0cbd488d763969ed965af8e19078
image
Captura desde 2022-08-25 12-36-23
Captura desde 2022-08-25 12-35-45
Captura desde 2022-08-25 12-41-34

You can check the same examples through the «Ubicación» box and selecting, for example, Q56064929, or any other WD item.

**Review by njrzr — APPROVED** [review]

**Comment by olea** [comment]
Verificando 957bac32e64724f15f0b4ece4ccf0ff1fff1226c... todo parece correcto :-)

**Pull request merged**